src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -141,4 +142,13 @@ void regmodule_offline_transformations(py::module m) {
             manager.run_passes(model);
         },
         py::arg("model"));
+
+    m_offline_transformations.def(
+        "apply_flush_fp32_subnormals_to_zero",
+        [](std::shared_ptr<ov::Model> model) {
+            ov::pass::Manager manager;
+            manager.register_pass<ov::pass::FlushFP32SubnormalsToZero>();
+            manager.run_passes(model);
+        },
+        py::arg("model"));
```

Comment:
not needed here to create a new Python routine, just update existing `m_offline_transformations` with this transformation as we did for `SmartReshape` and `CF`

Reply:
done